variables.tf

```diff
@@ -14,6 +16,14 @@ variable "tags" {
   default     = {}
 }
 
+variable "lock_resources" {
+  description = "Enables a delete lock on the resource group to prevent accidental deletions."
+  type        = bool
+  default     = true
+}
+
+#
```

Comment:
Comment?

Reply:
Preparation for separating the different groups of configurations